scripts/airgapped/deploy-1.9.sh

```diff
@@ -0,0 +1,180 @@
+webhook: $(img knative-releases/knative.dev/serving/cmd/webhook)
+autoscaler-hpa: $(img knative-releases/knative.dev/serving/cmd/autoscaler-hpa)
+net-istio-controller/controller: $(img knative-releases/knative.dev/net-istio/cmd/controller)
+net-istio-webhook/webhook: $(img knative-releases/knative.dev/net-istio/cmd/webhook)
+queue-proxy: $(img knative-releases/knative.dev/serving/cmd/queue)
+migrate: $(img knative-releases/knative.dev/pkg/apiextensions/storageversion/cmd/migrate)
+"
+
+juju deploy --trust --debug ./$(charm kserve-controller) --resource kserve-controller-image=$(img kserve-controller) --resource kube-rbac-proxy-image=$(img kubebuilder/kube-rbac-proxy) --config custom_images="configmap__agent: '$(img charmedkubeflow/agent)'
+configmap__batcher: '$(img charmedkubeflow/agent)'
+configmap__explainers__art: '$(img kserve/art-explainer)'
+configmap__logger: '$(img charmedkubeflow/agent)'
+configmap__router: '$(img kserve/router)'
+configmap__storageInitializer: '$(img charmedkubeflow/storage-initializer)'
+serving_runtimes__huggingfaceserver: '$(img kserve/huggingfaceserver)'
+serving_runtimes__lgbserver: '$(img charmedkubeflow/lgbserver)'
+serving_runtimes__kserve_mlserver: '$(img seldonio/mlserver)'
+serving_runtimes__paddleserver: '$(img charmedkubeflow/paddleserver)'
+serving_runtimes__pmmlserver: '$(img charmedkubeflow/pmmlserver)'
+serving_runtimes__sklearnserver: '$(img charmedkubeflow/sklearnserver)'
+serving_runtimes__tensorflow_serving: '$(img tensorflow/serving)'
+serving_runtimes__torchserve: '$(img pytorch/torchserve-kfs)'
+serving_runtimes__tritonserver: '$(img nvidia/tritonserver)'
+serving_runtimes__xgbserver: '$(img charmedkubeflow/xgbserver)'
+"
+
+juju deploy --trust --debug ./$(charm kubeflow-dashboard) --resource oci-image=$(img dashboard)
+juju deploy --trust --debug ./$(charm kubeflow-profiles) --resource profile-image=$(img profile-controller) --resource kfam-image=$(img kfam)
+juju deploy --trust --debug ./$(charm kubeflow-roles)
+juju deploy --trust --debug ./$(charm kubeflow-volumes) --resource oci-image=$(img volumes-web-app)
+juju deploy --trust --debug ./$(charm metacontroller-operator) --config metacontroller-image=$(img metacontroller)
+juju deploy --trust --debug ./$(charm mlmd) --resource oci-image=$(img ml_metadata_store_server)
+
+juju deploy --debug ./$(charm minio) --resource oci-image=$(img minio)
+juju deploy --trust --debug ./$(charm oidc-gatekeeper) --resource oci-image=$(img oidc-authservice)
+juju deploy --trust --debug ./$(charm pvcviewer-operator) --series=focal --resource oci-image=$(img pvcviewer-controller)
+
+juju deploy --trust --debug ./$(charm tensorboard-controller) --resource tensorboard-controller-image=$(img tensorboard-controller)
+juju deploy --trust --debug ./$(charm tensorboards-web-app) --resource tensorboards-web-app-image=$(img tensorboards-web-app)
+juju deploy --trust --debug ./$(charm training-operator) --config training-operator-image=$(img training-operator)
```

Comment:
We need to cherry-pick https://github.com/canonical/training-operator/pull/193 to `track/1.8` for the image to be fetched for 1.9/stable

Reply:
I created [this PR](https://github.com/canonical/training-operator/pull/194) for it